Airports/EGLK/SMR/Geo.txt

```diff
@@ -0,0 +1,34 @@
+;Start Blackbushe (EGLK) Geo

```

Comment:
the ;start  and ;end lines can be deleted

Reply:
(in all 3 files)

Reply:
Should all be complete